yt_dlp/extractor/pbs.py

```diff
@@ -692,6 +693,7 @@ def extract_redirect_urls(info):
                 subtitles.setdefault('en', []).append({
                     'url': caption_url
                 })
+        subtitles = self._merge_subtitles(subtitles, hls_subs)
```

Comment:
I believe duplicates are handled by this

Reply:
```
[pbs] Downloading JSON metadata
[pbs] how-fantasy-reflects-our-world-picecq: Downloading webpage
[pbs] how-fantasy-reflects-our-world-picecq: Downloading widget/partnerplayer page
[pbs] how-fantasy-reflects-our-world-picecq: Downloading portalplayer page
[pbs] how-fantasy-reflects-our-world-picecq: Downloading 0 video url info
[pbs] how-fantasy-reflects-our-world-picecq: Downloading m3u8 information
[pbs] how-fantasy-reflects-our-world-picecq: Downloading 1 video url info
Available subtitles for 3015504031:
Language Formats
en       vtt, sami, dfxp, srt, vtt
```

Reply:
the 2 `vtt`s must have different URLs. No way to avoid this atm. Just let it be